modules/core/react/lib/radius.ts

```diff
@@ -0,0 +1,17 @@
+export interface CanvasBorderRadiusNumber {
```

Comment:
Let's change this to `CanvasBorderRadius`. The `Number` is confusing, when most are technically strings